lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -17,11 +17,38 @@ defmodule TrentoWeb.V1.SapSystemView do
     database
     |> Map.from_struct()
     |> Map.delete(:__meta__)
+    |> Map.delete(:deregistered_at)
     |> Map.put(:database_instances, rendered_database_instances)
     |> add_system_replication_status_to_secondary_instance
   end
 
   def render("database_instance.json", %{instance: instance}) do
+    instance
+    |> Map.from_struct()
+    |> Map.delete(:__meta__)
+    |> Map.delete(:absent_at)
```

Comment:
We shouldn't remove this field. It is used in the frontend.
The error must be that it is not defined in the schema.